packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/index.js

```diff
@@ -16,6 +16,15 @@ angular
     angularTranslate,
     'oui',
   ])
-  .component('packXdslAccess', access);
+  .component('packXdslAccess', access)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(/* @ngTranslationsInject:json ./comfortExchange/translations */)
```

Comment:
Can't all these files be loaded in the dedicated module instead of main module?

Reply:
Yes I could do, but for the moment, dedicated modules are not been refactored. I will do that in a future task, so for the moment, translation injections are in the main module and will be injected in dedicated module later.